cmd/commandfuncs.go

```diff
@@ -68,6 +69,9 @@ func cmdStart(fl Flags) (int, error) {
 	if startCmdConfigFlag != "" {
 		cmd.Args = append(cmd.Args, "--config", startCmdConfigFlag)
 	}
+	if startCmdEnvfileFlag != "" {
+		cmd.Args = append(cmd.Args, "--envfile", startCmdEnvfileFlag)
```

Comment:
Hmm, this isn't working because `startCmdEnvfileFlag` is a boolean and not a string.

Reply:
sorry,i've fixed `startCmdEnvfileFlag`  to string